cirq-core/cirq/ops/three_qubit_gates.py

```diff
@@ -217,6 +217,55 @@ def controlled(
         )
 
 
+def decompose_all_to_all_connect_ccz_gate(
```

Comment:
Move this decomposition and corresponding test to `cirq-ionq/cirq_ionq/ionq_gateset.py`.

Reply:
Done